package.json

```diff
@@ -160,7 +160,7 @@
     "eslint-plugin-react-hooks": "^1.6.0",
     "eslint-plugin-scanjs-rules": "^0.1.4",
     "eslint-plugin-sonarjs": "^0.5.0",
-    "eslint-plugin-va-enzyme": "./script/custom-eslint/va-enzyme",
+    "eslint-plugin-va": "./script/eslint-plugin-va",
```

Comment:
We can probably delete the `custom-eslint` directory, right?

Reply:
yes! I'll do that